### PR TITLE
Remove ORC encryption test from 321cdh

### LINKED ISSUE #5785: [BUG]Tests module build failed in OrcEncryptionSuite for 321cdh
**Describe the bug**
Nightly build failed by below error.
[2022-06-08T05:10:50.374Z] [INFO] --- scala-maven-plugin:4.3.0:testCompile (scala-test-compile-first) @ rapids-4-spark-tests_2.12 ---
[2022-06-08T05:10:52.898Z] [INFO] Using incremental compilation using Mixed compile order
[2022-06-08T05:10:52.898Z] [INFO] Compiler bridge file: /root/.sbt/1.0/zinc/org.scala-sbt/org.scala-sbt-compiler-bridge_2.12-1.3.1-bin_2.12.15__52.0-1.3.1_20191012T045515.jar
[2022-06-08T05:10:52.898Z] [INFO] Compiling 85 Scala sources and 1 Java source to /home/jenkins/agent/workspace/jenkins-rapids_nightly-pre_release-github-391/tests/target/spark321cdh/test-classes ...
[2022-06-08T05:10:57.065Z] [ERROR] [Error] /home/jenkins/agent/workspace/jenkins-rapids_nightly-pre_release-github-391/tests/src/test/320+/scala/com/nvidia/spark/rapids/OrcEncryptionSuite.scala:25: object CryptoUtils is not a member of package org.apache.orc.impl
[2022-06-08T05:10:57.065Z] [ERROR] [Error] /home/jenkins/agent/workspace/jenkins-rapids_nightly-pre_release-github-391/tests/src/test/320+/scala/com/nvidia/spark/rapids/OrcEncryptionSuite.scala:35: not found: value CryptoUtils
[2022-06-08T05:10:57.065Z] [ERROR] [Error] /home/jenkins/agent/workspace/jenkins-rapids_nightly-pre_release-github-391/tests/src/test/320+/scala/com/nvidia/spark/rapids/OrcEncryptionSuite.scala:36: value AES_CTR_128 is not a member of object org.apache.orc.EncryptionAlgorithm
[2022-06-08T05:10:57.323Z] [ERROR] [Error] /home/jenkins/agent/workspace/jenkins-rapids_nightly-pre_release-github-391/tests/src/test/320+/scala/com/nvidia/spark/rapids/OrcEncryptionSuite.scala:25: Unused import
[2022-06-08T05:11:15.370Z] [ERROR] four errors found

### PR DESCRIPTION
**Pull request by razajafri**
Looks like 321cdh is using ORC 1.5.1 and ORC encryption is supported in 1.6+ therefore we will remove the test from 321cdh until we find out if they support ORC encryption 

fixes #5785 

Signed-off-by: Raza Jafri <rjafri@nvidia.com>

### CODE REVIEW

File: tests/pom.xml [hunk 1]
```diff
@@ -148,25 +148,6 @@
...
-                    <plugin>
-                        <groupId>org.codehaus.mojo</groupId>
-                        <artifactId>build-helper-maven-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <id>add-321cdh-test-src</id>
-                                <goals><goal>add-test-source</goal></goals>
-                                <configuration>
-                                    <sources>
-                                        <source>${project.basedir}/src/test/320+/scala</source>
```
Comment (tgravescs) [inline tests/pom.xml:162]: perhaps we should move this directory to be 320+-noncdh to match our source code

Comment (razajafri) [reply tests/pom.xml:162]: How about 320+-noncdh-nondb as it's not testing db either

Comment (tgravescs) [reply tests/pom.xml:162]: thatsfine

**Review by tgravescs — APPROVED** [review]

**Comment by tgravescs** [comment]
build

**Pull request merged**